### ISSUE TITLE
Support for NUnit TestContext.AddFormatter

### ISSUE THREAD

**Issue by jakob-ledermann** [fscheck/FsCheck#595]
Would it be possible to support NUnits ValueFormatters configured with TestContext.AddFormatter für FsCheck.Nunit tests?

**Comment by kurtschelfthout** [comment]
Could you give an example of what you’d like to see working, but doesn’t currently?

Also to manage expectations: on FsCheck.NUnit I am happy to take PRs but unlikely to write any new features myself.

**Comment by kurtschelfthout** [comment]
Yes I think this should be doable - again caveat my knowledge about NUnit is limited - but it seems like that should be a matter of routing FsCheck's output through NUnit's output system somehow: https://github.com/fscheck/FsCheck/blob/master/src/FsCheck.NUnit/FsCheckPropertyAttribute.fs#L25 

We do a similar thing for FsCheck.Xunit, which has a `ITestOutputHelper` we write to.
